### PR TITLE
Fix mypy test failure

### PR DESCRIPTION
**Pull request by annehaley**

### CODE REVIEW

File: miqa/core/rest/frame.py [hunk 1]
```diff
@@ -125,13 +125,21 @@ def create(self, request, *args, **kwargs):
             scan.save()
         elif 'scan' in serializer.data:
             scan = Scan.objects.get(id=serializer.data['scan'])
+            if not scan:
+                raise APIException(
+                    'Could not create new Frame.'
```
Comment (zachmullen) [inline miqa/core/rest/frame.py]: This should probably have status=404

File: miqa/core/rest/frame.py [hunk 2]
```diff
@@ -125,13 +125,21 @@ def create(self, request, *args, **kwargs):
...
             if not get_perms(request.user, scan.experiment.project):
                 Response(status=status.HTTP_403_FORBIDDEN)
         else:
             raise APIException(
                 'Provide either a parent scan or grandparent experiment for this frame.'
             )
 
+        if not scan:
+            raise APIException(
+                'Could not create new Frame.'
```
Comment (zachmullen) [inline miqa/core/rest/frame.py]: Same here

File: miqa/core/rest/frame.py (hunk 1 above)
Comment (zachmullen) [reply miqa/core/rest/frame.py]: Also, we should probably make the message more informative, like "Scan not found"

File: miqa/learning/nn_training.py [hunk 3]
```diff
@@ -588,7 +588,7 @@ def train_and_save_model(df, count_train, save_path, num_epochs, val_interval, o
         logger.info('-' * 25)
         logger.info(f'epoch {epoch + 1}/{num_epochs}')
         model.train()
-        epoch_loss = 0
+        epoch_loss = float(0)
```
Comment (zachmullen) [inline miqa/learning/nn_training.py]: Another way to write this is as a literal, `0.0`, which would avoid the cast

Comment (annehaley) [reply miqa/learning/nn_training.py]: Changed in [107d673](https://github.com/OpenImaging/miqa/pull/634/commits/107d673714769932d133942ea8355def62151cc7)

File: miqa/core/rest/frame.py (hunk 2 above)
Comment (annehaley) [reply miqa/core/rest/frame.py]: Changed in [1d6a5b5](https://github.com/OpenImaging/miqa/pull/634/commits/1d6a5b53988507eca80fd0184717ac0fbd3534e4)

File: miqa/core/rest/frame.py (hunk 1 above)
Comment (annehaley) [reply miqa/core/rest/frame.py]: Changed in [1d6a5b5](https://github.com/OpenImaging/miqa/pull/634/commits/1d6a5b53988507eca80fd0184717ac0fbd3534e4)

**Review by zachmullen — APPROVED** [review]

**Pull request merged**